know/rdf/queries/wikidata/get_landmark.sparql

```diff
@@ -0,0 +1,11 @@
+# Replace $0 with name entry 
+SELECT ?name ?description ?TerritoryLocation ?TerritoryLocationLabel ?CountryLocation ?CountryLocationLabel ?inception ?coordinates WHERE {
```

Comment:
Change these to camel case and remove the entries that aren't a label/exact value (e.g. `TerritoryLocation` rather than `TerritoryLocationLabel`)